servers/store/src/main/proto/table.proto

```diff
@@ -34,8 +34,11 @@ message IcebergMetadataPointer {
   string metadata_location = 1;
 }
 
-message IcebergGlobal {
-  string id_generators = 1;
+message IcebergRefState {
```

Comment:
In the contentAPI descriptions and some other place of code, It still mentions that on-ref-state is just the snapshot id. can we change that as well?
please look up `Iceberg: snapshot-ID` in the files.

Reply:
addressed